src/tokenize.cpp

```diff
@@ -11,6 +11,7 @@
 #include "char_table.h"
 #include "prototypes.h"
 #include "chunk_list.h"
+#include <stdbool.h>
```

Comment:
Hmm, why do you need this? 

The build fails because:
> ./scripts/Run_uncrustify_for_sources.sh; fi" exited with 1

Your code formatting somewhere does not align with the standards of this repo. Use Run_uncrustify_for_sources.sh to find out which files are missformatted and Uncrustify them with the forUncrustifySources.cfg (in the project dir) config.